software/software.mdown

```diff
@@ -5,12 +5,13 @@ JMAP is still young but several implementations already exist, and more are on t
 ## Clients
 
 * **[JMAP Demo Webmail](https://github.com/jmapio/jmap-demo-webmail)** (JavaScript, MIT): a sophisticated demo webmail client to be used as a base for new projects. Default client for the [proxy](https://proxy.jmap.io).
-* **[JMAP::Tester](https://metacpan.org/pod/JMAP::Tester)** (Perl, Perl5): a JMAP client made for testing JMAP servers
+* **[JMAP::Tester](https://metacpan.org/pod/JMAP::Tester)** (Perl, Perl5): a JMAP client made for testing JMAP servers.
+* **[NeoMutt](https://github.com/robn/neomutt/tree/jmap)** (C): A command-line mail reader. It's a version of Mutt with added features. An experimental fork with WIP JMAP support is available.
```

Comment:
I'd prefer not to mention this fork. It was an experiment that I am unlikely to continue with (at least not before a proper JMAP library for C exists).

Reply:
@robn Okay. I'm sorry. You're right. But the Cyrus link didn't work and I updated it. Sorry if I broke it. You can update the Cyrus Mail link and close this :)